packages/test/src/workflows/stack-tracer.ts

```diff
@@ -3,6 +3,7 @@
  * @module
  */
 import * as wf from '@temporalio/workflow';
+import { EnhancedStackTrace } from '@temporalio/workflow/src/interfaces';
```

Comment:
```suggestion
import type { EnhancedStackTrace } from '@temporalio/workflow/src/interfaces';
```

Reply:
I think there's an eslint rule that can verify this, don't think it's critical but don't mind changing either.

Reply:
Right. It might be worth adding the linter for this. Definitely not critical in this case.